docs/DeveloperGuide.adoc

```diff
@@ -242,6 +247,104 @@ _{Explain here how the data encryption feature will be implemented}_
 
 // end::dataencryption[]
 
+=== [Proposed] Schedule optimisation
+==== Proposed Implementation
+
+The proposed schedule optimisation feature utilises `TimeTable` and allows the user to schedule multiple activities within the same time slot. The following command is supported:
+
+* optimise budget INDEX_OF_DAY -- optimises and brings activities with the lowest cost into focus. This operation supports activities with or without a cost. Activities without a cost has a default cost of $0.00.
+
+
+Given below is a sample usage of the proposed feature:
+
+Step 1. User adds an activity with a cost:
+
+* `add activity n/lunch a/KFC c/5.00`
+
+Step 2. User adds another activity without a cost:
+
+* `add activity n/go for a walk a/park`
+
+Step 3. User schedules the first activity into a day:
+
+* `schedule activity 1 st/1300 et/1400 d/1`
+
+Step 4. User schedules the second activity into the same day:
+
+* `schedule activity 2 st/1300 et/1400 d/1`
+
+Step 6. User executes `optimise budget` command. The activities in that day would be shown in a way such that the activities with the lowest costs are in focus.
+
+Shown below is a summary of the execution of the command.
+
+image::OptimiseSequenceDiagram.png[]
+
+==== Design considerations
+Aspect: How version comparison is executed:
+
+* Alternative 1(current choice): user overload each time slot.
```

Comment:
Would be good if you could add justification of why you finally chose this alternative over the other one. Since both have pros and cons, how did you weigh them?